Tests/Common/Utils/XCTestCase+SupportedProtocols.swift

```diff
@@ -15,13 +15,17 @@ extension XCTestCase {
                 _ testCase: (TransportProtocol, XCTestExpectation) throws -> Void
     ) rethrows {
         let supportedProtocols = TransportProtocol.supportedProtocols
+
         let last = supportedProtocols.last
         try supportedProtocols.forEach { transportProtocol in
             let description = "[\(transportProtocol.description)]:\(description)"
             print("Testing ... \(description)")
             let expect = expectation(description: description)
             try testCase(transportProtocol, expect)
-            waitForExpectations(timeout: timeout)
+            XCTWaiter().wait(for: [expect], timeout: timeout)
```

Comment:
this is not any different, it still waits on one expectation

Reply:
But it doesn't care which one...   Here's what I was seeing:

Reply:
* http test runs
* wait for expectation for http test times out (b/c server n/a), but http process still in flight
* grpc test runs
* http test finally fulfills expectation due to failure
* grpc test fails, fulfilling expectation

Reply:
Apple recommends being explicit in the expectations being waited for - https://developer.apple.com/documentation/xctest/xctestcase/1500748-waitforexpectations

Reply:
resolving